webpack.config.js

```diff
@@ -31,7 +31,8 @@ module.exports = {
         test: /\.jsx?$/,
         include: [
           path.join(__dirname, 'client/'),
-          path.join(__dirname, 'common/')
+          path.join(__dirname, 'common/'),
+          path.join(__dirname, 'server/')
```

Comment:
I see the issue now. Yes we should be including server files in webpack's loaders. Instead we can relocate those utils into the common directory.